### PR TITLE
Send e-mail change order status

### PR DESCRIPTION
**Pull request by wallaceSF**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| **Issue**         | https://mundipagg.atlassian.net/browse/MOD-889
| **What?**         | Implement send e-mail when order status is changed
| **Why?**          |Some clients questioned not triggering e-mail
| **How?**          | It's important have sendmail on the server

### CODE REVIEW

File: src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php [hunk 1]
```diff
@@ -20,10 +20,10 @@ public function __construct()
         $this->logService = new OrderLogService();
     }
 
-    public function addHistoryComment($message)
+    public function addHistoryComment($message, $sendCustomerNotified = false)
```
Comment (michelpl) [inline src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php]: Mudar variável para $notifyCustomer

File: src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php [hunk 2]
```diff
@@ -20,10 +20,10 @@ public function __construct()
...
     {
         $message = 'MP - ' . $message;
-        $this->addMPHistoryComment($message);
+        $this->addMPHistoryComment($message, $sendCustomerNotified);
```
Comment (michelpl) [inline src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php]: Aqui também.

File: src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php [hunk 3]
```diff
@@ -190,7 +190,7 @@ public function getTotalPaidFromCharges()
         return $grandTotal;
     }
 
-    abstract protected function addMPHistoryComment($message);
+    abstract protected function addMPHistoryComment($message, $sendCustomerNotified);
```
Comment (michelpl) [inline src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php]: Aqui também

File: src/Kernel/Interfaces/PlatformOrderInterface.php [hunk 4]
```diff
@@ -22,7 +22,7 @@ public function setState(OrderState $state);
     public function setStatus(OrderStatus $status);
     public function getStatus();
     public function loadByIncrementId($incrementId);
-    public function addHistoryComment($message);
+    public function addHistoryComment($message, $sendCustomerNotified);
```
Comment (michelpl) [inline src/Kernel/Interfaces/PlatformOrderInterface.php]: Aqui também

**Review by michelpl — APPROVED** [review]

File: src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php (hunk 2 above)
Comment (michelpl) [reply src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php]: $notifyCustomer

File: src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php (hunk 3 above)
Comment (michelpl) [reply src/Kernel/Abstractions/AbstractPlatformOrderDecorator.php]: $notifyCustomer

**Review by michelpl — CHANGES_REQUESTED** [review]

**Review by GabrielDeveloper — DISMISSED** [review]

**Review by GabrielDeveloper — APPROVED** [review]

**Review by michelpl — APPROVED** [review]

**Pull request merged**